spec/models/manageiq/providers/hawkular/middleware_manager/refresher_spec.rb

```diff
@@ -39,9 +39,10 @@
     expect(@ems_hawkular.middleware_datasources).not_to be_empty
     expect(@ems_hawkular.middleware_messagings).not_to be_empty
     expect(@ems_hawkular.middleware_deployments.first.status).to eq('Enabled')
+    # See https://github.com/hawkular/hawkular-services/pull/258
     expect(@ems_hawkular.middleware_servers.first.properties).to include(
-      'Availability'            => 'up',
-      'Calculated Server State' => 'running'
+      'Availability'            => 'Running',
+      'Calculated Server State' => 'reload-required'
```

Comment:
This is weird.

Reply:
It is sort of a bug in the agent i guess, that does some changes but never reloads